responses/tests/test_responses.py

```diff
@@ -822,8 +844,11 @@ def send(
             verify=True,
             cert=None,
             proxies=None,
+            **kwargs,
```

Comment:
Niquests adapters have a few more arguments. So this is mainly for the compat purposes. By my reading, it does not affect your initial test.